src/main/java/org/aist/aide/mappingservice/domain/models/Mapping.java

```diff
@@ -71,24 +70,31 @@ public void setType(String type) {
         this.type = type;
     }
 
-    public void setAbstraction(String abstraction) {
-        this.abstraction = abstraction;
+    public void setDefaultAbstraction(String defaultAbstraction) {
+        this.defaultAbstraction = defaultAbstraction;
     }
 
-    public void setService(Service service) {
-        this.service = service;
+    public void upsertClassifier(Classifier classifier) {
```

Comment:
I don't think this logic belongs in a data class, maybe move it to the service?

Reply:
Ok